docs/essentials/launcher.md

```diff
@@ -92,6 +92,12 @@ await Launcher.OpenAsync(new OpenFileRequest
 });
 ```
 
+## Presentation Location When Opening Files
+
+![Pre-release API](~/media/shared/preview.png)
```

Comment:
Double checking that the preview image is required?

Reply:
I think it's not necessary

Reply:
I'm busy right now and can't delete this